polaris.shopify.com/src/components/IconsPage/IconsPage.tsx

```diff
@@ -1,310 +1,145 @@
+                  width={20}
+                  height={20}
+                  alt=""
+                />
+              </span>
+              <div>
+                <h2>No icon found</h2>
                 <p>
-                  <b>Minor icons</b>
+                  Open a{" "}
+                  <Link
+                    href={`https://github.com/Shopify/polaris/issues/new?title=[polaris.shopify.com] No icon found ${searchText}&labels=Icon`}
+                  >
+                    GitHub issue
+                  </Link>{" "}
+                  to send us feedback or propose new icons.
                 </p>
               </div>
-
-              <IconGrid>
-                {minorIcons.map((icon) => (
-                  <IconGrid.Item
-                    key={icon.name}
-                    icon={icon}
-                    onClick={() => getSelectedIcon(icon)}
-                  />
-                ))}
-              </IconGrid>
-            </>
-          )}
+            </div>
+          ) : null}
         </div>
-
-        {selectedIcon && (
-          <>
-            {useModal ? (
-              <Dialog open={modalIsOpen} onClose={() => setModalIsOpen(false)}>
-                <div className={styles.ModalBackdrop} aria-hidden="true" />
-
-                <Dialog.Panel className={styles.Modal}>
-                  <SidebarContent
-                    selectedIcon={selectedIcon}
-                    setFilterString={setFilterString}
-                    isInModal={true}
-                  />
-                </Dialog.Panel>
-              </Dialog>
-            ) : (
-              <SidebarContent
-                selectedIcon={selectedIcon}
-                setFilterString={setFilterString}
-                isInModal={false}
-              />
-            )}
-          </>
-        )}
-      </div>
-    </Container>
-  );
-}
-
-function SidebarContent({
-  selectedIcon,
-  setFilterString,
-  isInModal,
-}: {
-  selectedIcon: Icon;
-  setFilterString: (string: string) => void;
-  isInModal: boolean;
-}) {
-  const updateURL = `https://github.com/Shopify/polaris-icons/issues/new?assignees=@shopify/icon-guild&labels=Update,Proposal&template=propose-updates-to-existing-icons.md&title=%5BProposal%5D%20Update%20${selectedIcon.name}`;
-
-  return (
-    <div className={styles.Sidebar}>
-      <div className={className(styles.SidebarSection, styles.IconInfo)}>
-        <div className={styles.Preview}>
-          <div className={styles.PreviewImage}>
-            <Image
-              src={importedSvgs(`./${selectedIcon.fileName}.svg`)}
-              alt={metadata[selectedIcon.fileName].description}
-              width={20}
-              height={20}
-              icon
-            />
-          </div>
-          <div className={styles.IconSet}>{selectedIcon.set}</div>
-        </div>
-
-        <h2 className={styles.IconName}>
-          {isInModal ? (
-            <Dialog.Title>{selectedIcon.name}</Dialog.Title>
```

Comment:
Weird. I thought the semantic structure of the element would retain its usefulness. Some magic happening.

Reply:
> This is the title for your Dialog. When this is used, it will set the aria-labelledby on the Dialog.

I don't think this is required with a title tag. Testing with osx screen reader I didn't have any issues. I am going to leave as is for now.